src/sentry/static/sentry/app/views/settings/project/projectOwnership/modal.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {uniq} from 'lodash';
+import idx from 'idx';
+
+import {t} from '../../../../locale';
+import AsyncView from '../../../asyncView';
+
+import SentryTypes from '../../../../proptypes';
+import OwnerInput from './ownerInput';
+
+class Modal extends AsyncView {
+  static propTypes = {
+    organization: SentryTypes.Organization,
+    project: SentryTypes.Project,
+    issueId: PropTypes.string,
+  };
+
+  getEndpoints() {
+    let {organization, project, issueId} = this.props;
+    return [
+      ['ownership', `/projects/${organization.slug}/${project.slug}/ownership/`],
+      ['urlTagData', `/issues/${issueId}/tags/url/`],
+      ['eventData', `/issues/${issueId}/events/latest/`],
+    ];
+  }
+
+  renderBody() {
+    let {ownership, urlTagData, eventData} = this.state;
+
+    let urls = urlTagData.topValues.map(i => i.value);
+    let paths = uniq(
```

Comment:
this will throw on groups without exceptions, and i guess there's also a stacktrace entry here I should make sure it works on